src/lib/attach-action-provider.ts

```diff
@@ -0,0 +1,56 @@
+import { Action, StateContext } from '@ngxs/store';
+import { ActionHandlerMetaData } from '@ngxs/store/src/actions/symbols';
+import { MetaDataModel } from '@ngxs/store/src/internal/internals';
+import * as lodash from 'lodash';
+
+const META_KEY = 'NGXS_META';
+const META_OPTIONS_KEY = 'NGXS_OPTIONS_META';
```

Comment:
as these `consts` are now used from different providers, we should extract them to some `const.ts`. Please also add the comment from `attach-action.ts` that those values are selected from the core-codebase.